### ISSUE TITLE
[Feature] Switch to a monorepo 

### ISSUE THREAD

**Issue by hildjj** [peggyjs/peggy#60]
Several packages in a pkg directory (it conflicts less often with package.json when doing completions on the command line than packages)

- peggy (always/only published as a minified UMD which is tested for commonjs, es, ts, browser-script-src, and browser-import)
- peggy-cli (which depends on peggy and commander, and is node12+ starting in 2 weeks)
- sample programs (ts, etc), each marked `private` in their `package.json`

**Comment by StoneCypher** [comment]
The thing that bothered me the most about Futagoza was that he kept entering and exiting monorepos

My fork, instead of a monorepo, was a single repo.  That is, no weird side-chain tools, none of that mess

This stack is simple enough that there's just no good reason for any of the side stuff.  

If you'd allow it, I'd write a few PRs to show how much simpler this could all be, using standard node tooling

Basically every `lerna` stack dies

**Comment by hildjj** [comment]
I'm almost done, and it doesn't need lerna.  It's simplified a lot.  I'll not land the PR until you've had a chance to look at it and comment.
